src/main/java/no/nav/familie/integrasjoner/personopplysning/PersonopplysningerService.kt

```diff
@@ -76,8 +76,28 @@ class PersonopplysningerService(private val personSoapClient: PersonSoapClient,
         }
     }
 
+    fun hentDødsfall(personIdent: String, tema: String): DødsfallResponse {
+        val doedsfall = pdlRestClient.hentDødsfall(personIdent, tema)
+        return DødsfallResponse(erDød = doedsfall.isNotEmpty(),
+                                dødsdato = doedsfall.filter { it.doedsdato != null }
+                                        .map { it.doedsdato }
+                                        .firstOrNull())
+    }
+
+    fun harVerge(personIdent: String, tema: String): VergeResponse {
+        val harVerge =
+                pdlRestClient.hentVergemaalEllerFremtidsfullmakt(personIdent, tema)
+                        .filter { it.type != "stadfestetFremtidsfullmakt" }
+                        .isNotEmpty()
```

Comment:
Her er det mulig å forenkle med
`.any { it.type != "stadfestetFremtidsfullmakt" }`

Reply:
Det går an!